.husky/pre-commit

```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
what does this git command achieve?

Reply:
it gives the current branch name 
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/11079632/199018853-7bd9a7b9-59ae-43fa-bb0d-95f5a0e61c56.png">

Reply:
got it, thanks!